Tests/configure_and_test_integration_instances.py

```diff
@@ -757,7 +758,14 @@ def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci
         'marketplace.initial.sync.delay': '0',
         'content.pack.ignore.missing.warnings.contentpack': 'true'
     }
-    if not is_nightly:
+    if is_private:
+        server_configuration['marketplace.bootstrap.bypass.url'] = 'https://storage.googleapis.com/marketplace-ci-build'
+        server_configuration['marketplace.gcp.path'] = 'content/builds/{}/{}/content/packs'.format(branch_name,
```

Comment:
Same question about the location of `dummy` index. I remember that we discussed on updating same index for private packs testing on static location. I probably miss something